------------------------------------------------------------
revno: 13229
revision-id: squid3@treenet.co.nz-20151119051038-vwclso11p68sgvxc
parent: squid3@treenet.co.nz-20150828132334-3r7v4kh1lrqprrqg
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.4
timestamp: Wed 2015-11-18 21:10:38 -0800
message:
  Fix delay_parameters documentation
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20151119051038-vwclso11p68sgvxc
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.4
# testament_sha1: 2531c5b2bc5fd88d4a875d2d4fa5750f41b0414b
# timestamp: 2015-11-19 05:12:06 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.4
# base_revision_id: squid3@treenet.co.nz-20150828132334-\
#   3r7v4kh1lrqprrqg
# 
# Begin patch
=== modified file 'src/cf.data.pre'
--- src/cf.data.pre	2015-01-24 05:13:20 +0000
+++ src/cf.data.pre	2015-11-19 05:10:38 +0000
@@ -6173,7 +6173,7 @@
 
 		delay_parameters 1 -1/-1 8000/8000
 
-	Note that 8 x 8000 KByte/sec -> 64Kbit/sec.
+	Note that 8 x 8K Byte/sec -> 64K bit/sec.
 
 	Note that the figure -1 is used to represent "unlimited".
 
@@ -6188,9 +6188,9 @@
 
 		delay_parameters 2 32000/32000 8000/8000 600/8000
 
-	Note that 8 x 32000 KByte/sec -> 256Kbit/sec.
-		  8 x  8000 KByte/sec ->  64Kbit/sec.
-		  8 x   600 Byte/sec  -> 4800bit/sec.
+	Note that 8 x  32K Byte/sec ->  256K bit/sec.
+		  8 x   8K Byte/sec ->   64K bit/sec.
+		  8 x 600  Byte/sec -> 4800  bit/sec.
 
 
 	Finally, for a class 4 delay pool as in the example - each user will

